### PR TITLE
feat: [BD-46] implement rtl for components from Collapsible to Dropdown

### PR DESCRIPTION
**Pull request by vlasovmichael**
- Implement RTL support for all component
- from Collapsible to Dropdown
- JIRA: https://openedx.atlassian.net/browse/PAR-646

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @vlasovmichael! I've created [BLENDED-1092](https://openedx.atlassian.net/browse/BLENDED-1092) to keep track of it in Jira.   More details are on the [BD-46](https://openedx.atlassian.net/wiki/spaces/COMM/pages/3134685276/BD-46+Paragon+Blended+Support+and+Maintenance) project page.


When this pull request is ready, tag your edX technical lead.

**Review by adamstankiewicz — COMMENTED** [review]
I think this is all set except for one thing I noticed with RTL on the `Card` component (see screenshot).

File: src/Card/Card.scss [hunk 1]
```diff
@@ -79,6 +79,13 @@
     justify-content: flex-end;
```
[the remark this answers is not in the document]
Comment (vlasovmichael) [reply src/Card/Card.scss:79]: It's not a bug. I implemented spacing in another PR. I have a local branch with all commits. Screen attached
image

**Review by adamstankiewicz — APPROVED** [review]

File: src/Card/Card.scss (hunk 1 above)
[the remark this answers is not in the document]
Comment (adamstankiewicz) [reply src/Card/Card.scss:82]: @vlasovmichael Ah, OK thanks for the clarification.

**Comment by edx-webhook** [comment]
@vlasovmichael 🎉 Your pull request was merged!  Please take a moment  to [answer a two question survey](https://docs.google.com/forms/d/e/1FAIpQLSceJOyGJ6JOzfy6lyR3T7EW_71OWUnNQXp68Fymsk3MkNoSDg/viewform?usp=pp_url&entry.1671973413=openedx/paragon&entry.867055334=https://github.com/openedx/paragon/pull/1082&entry.1484655318=https://github.com/vlasovmichael&entry.752974735=2022-02-10+11:04&entry.1917517419=2022-02-25+12:03&entry.2133058324=Yes) so we can improve your experience in the future.

**Pull request merged**